src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
+
+    protected String from;
+    protected String to;
+
+    public Event(String description, int num,String from, String to){
```

Comment:
```suggestion
    public Event(String description, int num, String from, String to) {
```
It may improve readability to standardize spacing according to [Google's Coding Standards](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace). I noticed the same issue in other places too.